bundles/org.openhab.core.addon.xml/src/main/java/org/openhab/core/addon/xml/internal/AddonInfoReader.java

```diff
@@ -68,11 +69,11 @@ protected void registerConverters(XStream xstream) {
 
     @Override
     protected void registerAliases(XStream xstream) {
-        xstream.alias("binding", BindingInfoXmlResult.class);
+        xstream.alias("addon", AddonInfoXmlResult.class);
         xstream.alias("name", NodeValue.class);
         xstream.alias("description", NodeValue.class);
         xstream.alias("author", NodeValue.class);
-        xstream.alias("service-id", NodeValue.class);
```

Comment:
What has happened to the `service-id`?

Reply:
Originally I planned to change it to `config-id` and later decided against that. I have reverted this changed (same answer below).